src/lib/components/Layout/Col.svelte

```diff
@@ -0,0 +1,38 @@
+﻿<div {...props}>
+  <slot />
+</div>
+
+<script lang="ts">
+  type colSpan = number | string | undefined;
```

Comment:
Use null instead of undefined
```suggestion
  type colSpan = number | string | null;
```